src/content/pages/sponsor/information.mdx

```diff
@@ -220,11 +224,12 @@ Here are shipping information provided by DB Schenker:
   - **This is the [sample PIPL](https://docs.google.com/spreadsheets/d/1gMajRAPFcdZ2V0Nsw-KKUZuMqGOa9JIs/view) provided (for NON-EU goods)**
   - **Please contact DB Schenker for more Details**
 
-- [Shipping Quotes (.doc)](https://docs.google.com/document/d/1_49ECWNMrFg7znFQjJ3AffSpyTZg7sAi/view)
+- [Shipping Quotes (.doc)](https://docs.google.com/document/d/1YLOdAEh409HeGeOojxjaNDYX5vZPqZvh)
   - **Contact DB Schenker directly to discuss pricing and conditions**
 
 **Deadlines**
-- Deadlines are specified in [Shipping Guidelines & Address (.doc)](https://docs.google.com/document/d/1vtydTQNkLDp2e4NrMg0VIhc2AYM8zMOK/view).
+- Deadline for reciving shipments is **1st of July**.
```

Comment:
Maybe we can explicitly say that DB Schenker should receive goods by 1st of July. What do you think?
```suggestion
- DB Schenker should receive shipments by **1st of July**.
```